chapters/ch58.md

```diff
@@ -1,17 +1,16 @@
 > # 58. Building a superhero team - Get your teammates to read this
 
---> _replace THIS LINE by your translation for the above line_
-
+# 58. Xây dựng một biệt đội siêu hùng mạnh - Hãy để đồng đội của bạn đọc điều này
 > Congratulations on finishing this book!
 
---> _replace THIS LINE by your translation for the above line_
+Chúc mừng bạn đã hoàn thành quyển sách này!
 
 > In Chapter 2, we talked about how this book can help you become the superhero of your team.
 
---> _replace THIS LINE by your translation for the above line_
+Trong chương 2, chúng tôi đã nói về việc quyển sách này có thể giúp bạn trở thành siêu anh hùng trong đội của bạn.
```

Comment:
* Từ "we" trong văn viết thường được dịch là "chúng ta", bao gồm cả người viết và người đọc.
* Từ "team" có thể dịch là "đội" hoặc "nhóm". Trong chương 2 đã dịch là "nhóm" nên mình cần thống nhất. Cả cụm "superhero team" vẫn có thể dịch là "biệt đội siêu anh hùng", nhưng từ "team" tách riêng đã được dịch là nhóm.